src/qsub.jl

```diff
@@ -68,7 +70,7 @@ function launch(manager::Union{PBSManager, SGEManager}, params::Dict, instances_
 end
 
 function manage(manager::Union{PBSManager, SGEManager}, id::Integer, config::WorkerConfig, op::Symbol)
-    if op == :finalize
+    if op == :interrupt
```

Comment:
The default implementation of `kill(manager, pid, config)` executes a `exit()` on the remote worker. So `rmprocs(workers())` should result in the killing of workers. Can you try it out?

If not, you can implement `kill(manager::Union{PBSManager, SGEManager}, id::Integer, config::WorkerConfig)` and manually manage the killing of workers as you are doing here.

`:finalize` is called when the `manager` objects is `gc`ed - the timing of which is not predictable.